src/materialized/src/bin/materialized/main.rs

```diff
@@ -359,14 +359,11 @@ pub struct Args {
     #[clap(long, env = "MZ_PERSIST_BLOB_URL")]
     persist_blob_url: Option<Url>,
     /// Where the persist library should perform consensus.
-    ///
-    /// Defaults to the `persist/consensus` SQLite database in the data
-    /// directory.
     #[clap(long, env = "MZ_PERSIST_CONSENSUS_URL")]
-    persist_consensus_url: Option<Url>,
+    persist_consensus_url: Url,
     /// Postgres catalog stash connection string.
     #[clap(long, env = "MZ_CATALOG_POSTGRES_STASH", value_name = "POSTGRES_URL")]
-    catalog_postgres_stash: Option<String>,
+    catalog_postgres_stash: String,
```

Comment:
This causes the cli test to fail because now the parameters are required, and one cannot simply pass `--version` alone anymore.